encoders/nlp/TransformerTFEncoder/__init__.py

```diff
@@ -145,9 +141,7 @@ def encode(self, data: 'np.ndarray', *args, **kwargs) -> 'np.ndarray':
         token_ids_batch = self.array2tensor(ids_info['input_ids'])
         mask_ids_batch = self.array2tensor(ids_info['attention_mask'])
         with self.no_gradients():
-            outputs = self.model(token_ids_batch,
-                                 attention_mask=mask_ids_batch,
-                                 output_hidden_states=True)
```

Comment:
is `output_hidden_states` not needed anymore?

Reply:
You can see that `output_hidden_states` is now set in `post_init`

Reply:
very good point ;)